.github/workflows/documentation.yaml

```diff
@@ -46,18 +46,14 @@ jobs:
             /tmp/doc.tar.gz
 
       - name: 🕵️ Check if the zsd docs need to be updated
-      # NOTE: We need to run git diff twice here since w/o --quiet it's RC
-      # is always 0. The first is to display the diff output, the second one
-      # to test whether the diff is empty.
         run: |
-          git --no-pager diff
-          if git --no-pager diff --quiet
+          if git --no-pager diff --exit-code
           then
             echo "✅ The zshelldocs are up to date"
           else
             {
               echo "❌ The zshelldocs in the repo are not up to date"
               echo 'Please regenerate them with $ make doc'
-            } >&2
```

Comment:
This wasn't intentional I assume.

Reply:
It was. It spits out the git diff output in all cases, allowing the omission of the second git diff command just to get the return code.

Reply:
I meant this line. Why not show the error message on stderr?